lib/StackItem.js

```diff
@@ -172,12 +172,12 @@ class StackItem extends Component<Props> {
 
   render() {
     const { position, stack, stackItem, zIndex } = this.props
-    const { transitionOptions } = getStackItemOptions(stackItem, stack)
+    const { transitionOptions, stackItemContainerStyle = {} } = getStackItemOptions(stackItem, stack)
```

Comment:
I'd suggest setting this default value directly within `getStackItemOptions.js` given that we're already checking quite a few possible values for it there.